CHANGELOG.md

```diff
@@ -39,7 +39,7 @@
   * Initial release
     FIXES [#3253](https://github.com/microsoft/Microsoft365DSC/issues/3253)
 * EXOQuarantinePolicy
-  * Support exporting global quarantine policy
+  * Support exporting and importing global quarantine policy
```

Comment:
Please add this under the unreleased section above.

Reply:
Updated